<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWifiInfoWidget</class>
 <widget class="QWidget" name="CWifiInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="UBXPushButton" name="pushButtonScan">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </widget>
  <widget class="CTableWidget" name="tableWidgetWifiInfos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHint">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connecting</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableWidget</class>
   <extends>QTableWidget</extends>
   <header>../zxLib/widget/ctablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>UBXPushButton</class>
   <extends>QPushButton</extends>
   <header>ubxpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
